frontend/context/BalanceProvider.tsx

```diff
@@ -51,16 +51,13 @@ export const BalanceContext = createContext<{
   totalEthBalance?: number;
   totalOlasBalance?: number;
   isLowBalance: boolean;
-  wallets?: Wallet[];
+  wallets?: Wallets[];
```

Comment:
shouldn't it be just
```suggestion
  wallets?: Wallets;
```
?